src/components/OSCALLoader.js

```diff
@@ -43,6 +43,9 @@ const EditorSplit = styled(Split)`
 `;
 
 export default function OSCALLoader(props) {
+  // THIS SHOULD BE PRINTING
+  console.log(props.renderForm, hasDefaultUrl);
+  
```

Comment:
As the comment suggests, this really should be logging something. I am not sure if it has something to do with render being asynchronous, not being able to create just a `OSCALLoader`, or something else entirely.

Reply:
This was solved by using `render`